services/ircd/default.nix

```diff
@@ -0,0 +1,406 @@
+let
+  ipAddress = "136.206.15.52";
```

Comment:
Hm, can you add to common/options.nix a variable for something like ircServerAddress?

Reply:
done